ravenpy/models/aggregate_forcings_to_HRUs.py

```diff
@@ -0,0 +1,232 @@
+from ravenpy.models.importers import (
+    RoutingProductGridWeightImporter,
+    GridWeightsCommand
+)
+import netCDF4 as nc4
+import numpy as np
+
+
+DIM_NAMES = ("lon", "lat")
+VAR_NAMES = ("lon", "lat")
+ROUTING_ID_FIELD = "HRU_ID"
+NETCDF_INPUT_FIELD = "NetCDF_col"
+AREA_ERROR_THRESHOLD = 0.05
+VARIABLES_TO_AGGREGATE = ["Streaminputs"]
+
+def convert_cell_id_to_ilat_ilon( gws, nlon, nlat ):
+    """
+    Purpose:
+       convert cell_id used in weights file (required by Raven) into 
+       latitude ID (ilat) and longitude ID (ilon);
+       both starting with 0
+
+    Return:
+      gridweights structure with (ilon, ilat) instead of (cell_id)
+
+    Arguments:
+      gws  (structure) ... grid-weights-structure as returned by RoutingProductGridWeightImporter and then extracted()
+      nlon (integer)   ... number longitudes NetCDF file
+      nlat (integer)   ... number latitudes  NetCDF file
```

Comment:
We're using https://numpydoc.readthedocs.io/en/latest/format.html for the docstring convention.